### DISCUSSION TITLE
希望添加一个单字搜寻功能

### DISCUSSION THREAD

**Discussion by ymml7** [rime/home#1618] [Ideas]
有的时候打多个字如果遇到单字很容易出现和后面的字组成词汇, 比如输入"是风动"的时候出现"是封懂"和"适逢", "侍奉"等. 如果我想要从结果列表里面找"是", 又需要较多的时间(我直接用的别人词库, 所以用不上的词汇比较多)

**Comment by yrljroli** [comment]
本来就支持。

在输入「shifengdong」时，单击左键两次，即可将光标移至「shi」与「feng」间，此时即只出现「shi」相应的字。

这一用法与常见输入法相同。

**Reply by yrljroli** [reply]
注：此「左键」指键盘上的向左（<kbd>←</kbd>）按键，非指滑鼠左键。

**Comment by wzv5** [comment]
自己写了一个：
https://github.com/rime/home/discussions/1756
